roles/bwc_repos/tasks/bwc_repos_apt.yml

```diff
@@ -0,0 +1,40 @@
+---
+
+- name: Assert that master_token is specified
+  fail: msg="License key must be supplied for BWC enterprise installation."
+  when: license is not defined
```

Comment:
Additionally, vars should follow the `{{ role_name}}_var_name` naming.
I guess for `bwc` role ideal var could be `{{ bwc_license }}`. And yeah, I must admit that for `bwc_repo` or `bwc_repos` role var name `{{ bwc_repo_license }}` looks a bit ugly :/